packages/edit-site/src/components/list/added-by.js

```diff
@@ -77,18 +81,26 @@ function AddedByAuthor( { id } ) {
 	] );
 	const [ isImageLoaded, setIsImageLoaded ] = useState( false );
 
+	const hasAvatar = !! user?.avatar_urls?.[ 48 ];
+
 	return (
 		<HStack alignment="left">
 			<div
-				className={ classnames( 'edit-site-list-added-by__avatar', {
+				className={ classnames( {
+					'edit-site-list-added-by__avatar': hasAvatar,
+					'edit-site-list-added-by__icon': ! hasAvatar,
```

Comment:
Maybe it could just be:

```suggestion
				className={ classnames(
					hasAvatar 
						? 'edit-site-list-added-by__avatar'
						: 'edit-site-list-added-by__icon',
				{
```